apps/core/scenes/search/Search.js

```diff
@@ -150,27 +27,8 @@ class Search extends React.Component<Props> {
           <View style={styles.illustrationWrapper}>
             <Illustration name="Boarding" style={styles.boardingIllustration} />
           </View>
-          <SearchFormModes />
-
-          <View style={desktopLayout && styles.inputsDesktop}>
-            <Placepickers />
-            <Datepickers />
-
-            <View
-              style={[styles.bottom, desktopLayout && styles.bottomDesktop]}
-            >
-              <Button
-                onPress={this.handleSubmitPress}
-                label="Search"
-                rightIcon={
-                  Platform.OS === 'web' ? <Icon name="chevron-right" /> : null
-                }
-              />
-            </View>
-          </View>
-          <View style={styles.bottomSpacer} />
+          <SearchForm />
```

Comment:
`onSubmit` web call stopped working. We will probably need to update and revise this part a bit.
But can you for now do just `<SearchForm onSubmit={this.props.onSubmit} />` to pass function down to make it work again (it is for now defined inside `web/pages/index.js` ).